src/main/java/ui/components/pickers/LabelPickerDialog.java

```diff
@@ -3,157 +3,256 @@
-            }
+    private final void populateAddedLabels(List<String> addedLabels, Optional<String> suggestion) {
+        addedLabels.stream()
+                .forEach(label -> {
+                    assignedLabels.getChildren().add(processAddedLabel(label, suggestion));
+                });
+    }
+
+    private final Node processAddedLabel(String addedLabel, Optional<String> suggestion) {
+        if (!suggestion.isPresent() || !addedLabel.equals(suggestion.get())) {
+            return getStyledPickerLabel(
+                new PickerLabel(TurboLabel.getFirstMatchingTurboLabel(allLabels, addedLabel), true));
         }
+        return getStyledPickerLabel(
+                new PickerLabel(TurboLabel.getFirstMatchingTurboLabel(allLabels, addedLabel), true)
+                .faded(true).removed(true));
     }
 
-    private void populateBottomBox(List<PickerLabel> bottomLabels, Map<String, Boolean> groups) {
-        bottomBox.getChildren().clear();
-        if (bottomLabels.isEmpty()) {
-            Label label = new Label("No labels in repository. ");
-            label.setPadding(new Insets(2, 5, 2, 5));
-            bottomBox.getChildren().add(label);
-        } else {
-            List<String> groupNames = groups.entrySet().stream().map(Map.Entry::getKey).collect(Collectors.toList());
-            Collections.sort(groupNames, String.CASE_INSENSITIVE_ORDER);
-
-            groupNames.stream().forEach(group -> {
-                Label groupName = new Label(group + (groups.get(group) ? "." : "-"));
-                groupName.setPadding(new Insets(0, 5, 5, 0));
-                groupName.setMaxWidth(ELEMENT_MAX_WIDTH - 10);
-                groupName.setStyle("-fx-font-size: 110%; -fx-font-weight: bold;");
-
-                FlowPane groupPane = new FlowPane();
-                groupPane.setHgap(5);
-                groupPane.setVgap(5);
-                groupPane.setPadding(new Insets(0, 0, 10, 10));
-                bottomLabels
-                        .stream()
-                        .filter(label -> label.getGroup().isPresent())
-                        .filter(label -> label.getGroup().get().equalsIgnoreCase(group))
-                        .forEach(label -> groupPane.getChildren().add(label.getNode()));
-                bottomBox.getChildren().addAll(groupName, groupPane);
-            });
-
-            FlowPane noGroup = new FlowPane();
-            noGroup.setHgap(5);
-            noGroup.setVgap(5);
-            noGroup.setPadding(new Insets(5, 0, 0, 0));
-            bottomLabels
-                    .stream()
-                    .filter(label -> !label.getGroup().isPresent())
-                    .forEach(label -> noGroup.getChildren().add(label.getNode()));
-            if (noGroup.getChildren().size() > 0) bottomBox.getChildren().add(noGroup);
+    private final void populateSuggestedLabel(List<String> addedLabels, Optional<String> suggestion) {
```

Comment:
I asked because 'populate' usually means fill something with multiple items. In this case, there is only one label. But it fine to keep it, to be consistent with other similar methods.